test/k8sT/Services.go

```diff
@@ -1548,6 +1549,132 @@ Secondary Interface %s :: IPv4: (%s, %s), IPv6: (%s, %s)`, helpers.DualStackSupp
 		})
 	})
 
+	SkipContextIf(func() bool {
+		// The graceful termination feature depends on enabling an alpha feature
+		// EndpointSliceTerminatingCondition in Kubernetes.
+		return helpers.RunsOnGKE() || helpers.RunsOnEKS()
+	}, "Checks graceful termination of service endpoints", func() {
+		const (
+			clientPodLabel = "app=graceful-term-client"
+			serverPodLabel = "app=graceful-term-server"
+			testPodLabel   = "zgroup=testDSClient"
+		)
+		var (
+			gracefulTermYAML string
+			clientPod        string
+			serverPod        string
+			testPodYAML      string
+		)
+
+		BeforeAll(func() {
+			if helpers.SkipK8sVersions("<1.20") {
```

Comment:
Could we move this to the SkipContextIf condition?

Reply:
I decided to keep it looking at other references in the code - it's useful to have the context why the test is being skipped.